kura/org.eclipse.kura.web2/pom.xml

```diff
@@ -1,19 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2011, 2016 Eurotech and others
+    Copyright (c) 2011, 2016 Eurotech and/or its affiliates and others
 
      All rights reserved. This program and the accompanying materials
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
 
-    Contributors:
-      Eurotech
-      Red Hat Inc - remove dependency on slf4j:logj12
-         - Fix build warning
-	  Amit Kumar Mondal (admin@amitinside.com)
```

Comment:
a) this seems to be aligned using tabs, not spaces
b) If we go for the Eclipse proposal this would be "Amit Kumar Mondal (Eurotech)"